### PR TITLE
PUBDEV-5210: Disable Deep Water from AutoML

### PR DESCRIPTION
**Pull request by navdeep-G**
* Disable for now as support for Deep Water is low.

### CODE REVIEW

File: h2o-automl/src/main/java/ai/h2o/automl/AutoML.java [hunk 1]
```diff
@@ -6,8 +6,8 @@
 import hex.ModelBuilder;
 import hex.StackedEnsembleModel;
 import hex.deeplearning.DeepLearningModel;
-import hex.deepwater.DeepWater;
-import hex.deepwater.DeepWaterParameters;
+//import hex.deepwater.DeepWater;
```
Comment (michalkurka) [inline h2o-automl/src/main/java/ai/h2o/automl/AutoML.java]: Can we just outright delete it? I don't think it will be ever needed

**Comment by michalkurka** [comment]
Looks good, please delete the code instead of commenting out

File: h2o-automl/src/main/java/ai/h2o/automl/AutoML.java (hunk 1 above)
Comment (navdeep-G) [reply h2o-automl/src/main/java/ai/h2o/automl/AutoML.java]: Yes, will delete.

**Comment by navdeep-G** [comment]
@mmalohlava I can do that in another PR or would you want it in this one?

**Comment by mmalohlava** [comment]
@navdeep-G I would do that in one shot.

**Pull request merged**